hdfs/src/main/java/io/ddf/hdfs/HDFSDDFManager.java

```diff
@@ -68,31 +70,24 @@ public DataFormat getDataFormat(HDFSDDF hdfsDDF) throws DDFException {
     if (hdfsDDF.getIsDir()) {
       try {
         FileStatus[] files = this.fs.listStatus(new Path(hdfsDDF.getPath()));
-        if (files.length == 0) {
-          throw new DDFException(String.format("There is no file under %s", hdfsDDF.getPath()));
-        }
+        // if (files.length == 0) {
```

Comment:
Our code style guide is not to have commented out block, we have git history when we want to check for old behavior, keep it in the code will be confusing